dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/CommandMapper.xml

```diff
@@ -46,6 +46,7 @@
     <select id="queryCommandPage" resultType="org.apache.dolphinscheduler.dao.entity.Command">
         select *
         from t_ds_command
-        order by update_time asc
+        order by process_instance_priority, id asc
```

Comment:
Does it change the behavior? The order cause order by  `update_time` but now order by `process_instance_priority`

Reply:
yes, we think priority is the more correct order than update_time after discussion.